packages/contracts-bedrock/test/kontrol/scripts/run-kontrol.sh

```diff
@@ -125,7 +125,7 @@ regen=
 test_list=()
 if [ "$SCRIPT_TESTS" == true ]; then
   test_list=( "OptimismPortalKontrol.prove_proveWithdrawalTransaction_paused0" \
-              "OptimismPortalKontrol.prove_proveWithdrawalTransaction_paused1" \
+              "OptimismPortalKontrol.prove_proveWithdrawalTransaction_paused1(" \
```

Comment:
Precisely, the point of this PR is to include the extra parenthesis to avoid the name of the test match with `OptimismPortalKontrol.prove_proveWithdrawalTransaction_paused10`

Reply:
This has been tested locally. No syntax errors should arise from restricting the name of the test that is matched